### PR TITLE
SEO and Site Setting

### PR DESCRIPTION
**Pull request by samarthsharma36**
@yvesh I have made the changes as per your suggestion :)
1. Search engine friendly URLs
2. Site offline/online
@puneet0191

### CODE REVIEW

File: src/JoomlaBrowser.php [hunk 1]
```diff
@@ -1169,31 +988,154 @@ public function createCategory($title, $extension = '')
...
+	 * @return  void
+	 *
+	 * @since   3.7.5
+	 */
+	public function setSiteOffline($enable)
+	{
+        $this->debug('I set the site to offline');
+        $this->doAdministratorLogin();
+        $this->amOnPage('/administrator/index.php?option=com_config');
+		if($enable == true){
```
Comment (yvesh) [inline src/JoomlaBrowser.php]: if ($enable)
{
   // No else needed this way
   return;
}

// .. else code

File: src/JoomlaBrowser.php [hunk 2]
```diff
@@ -1169,31 +988,154 @@ public function createCategory($title, $extension = '')
...
 		$this->checkForPhpNoticesOrWarnings();
 	}
+	/**
+	 * Site Offline
+	 *
+	 * @return  void
+	 *
+	 * @since   3.7.5
+	 */
+	public function setSiteOffline($enable)
```
Comment (yvesh) [inline src/JoomlaBrowser.php:1020]: default value true please - ($enable = true)

File: src/JoomlaBrowser.php [hunk 3]
```diff
@@ -1169,31 +988,154 @@ public function createCategory($title, $extension = '')
...
+	 *
+	 * @param   \AcceptanceTester   $I  Acceptance Tester
+	 *
+	 * @return void
+	 */
+	public function setSiteSearchEngineFriendly($enable)
+	{
+        $this->debug('I set the Search engine optimisation to Yes');
+        $this->doAdministratorLogin();
+        $this->amOnPage('/administrator/index.php?option=com_config');
```
Comment (yvesh) [inline src/JoomlaBrowser.php]: code style

**Comment by samarthsharma36** [comment]
Thanks for pointing that out @yvesh .
Can you review it and check if it is ready for merge? @puneet0191

File: src/JoomlaBrowser.php [hunk 4]
```diff
@@ -136,12 +125,11 @@ public function doAdministratorLogin($user = null, $password = null, $useSnapsho
 		$this->debug('I click Login button');
 		$this->click($this->locator->adminLoginButton);
 		$this->debug('I wait to see Administrator Control Panel');
-		$this->waitForText($this->locator->adminControlPanelText, 4, $this->locator->controlPanelLocator);
+		$this->waitForText('Control Panel', 4, $this->locator->controlPanelLocator);
```
Comment (puneet0191) [inline src/JoomlaBrowser.php:128]: why change this?

File: src/JoomlaBrowser.php [hunk 5]
```diff
@@ -1164,31 +988,154 @@ public function createCategory($title, $extension = '')
...
+        	$this->see('Select a Module Type');
+        	/**
+         	 * Module Types
+         	 * Articles - Archived
+         	 * Articles - Categories
+         	 * Articles - Category
+         	 */
+        	switch ($moduleType)
+        	{
+            	case 'Articles - Archived' :
```
Comment (puneet0191) [inline src/JoomlaBrowser.php]: I think the styling is all wrong here, also please make sure you only work with branch 4.0 on Joomla Browser, your PR should also be on 4.0, don't send PR for develop branch.

**Review by puneet0191 — CHANGES_REQUESTED** [review]
@Samgithub1997 please do the changes.

**Pull request closed without merging**